snapcraft/internal/meta/application.py

```diff
@@ -85,7 +85,7 @@ def can_use_wrapper(self, base: Optional[str]) -> bool:
         if self.command_chain:
             return False
 
-        # We only allow wrappers for core and core18.
+        # We only allow wrappers for core, core18 and core20.
```

Comment:
this is not desired, wrappers are supposed to be gone moving forward.

Reply:
Are the wrappers removed immediately?
Or is there a backward compatibility?